src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Vendor new SDK and bump API version to 2023-11-02-preview.
```

Comment:
Please also move this to your new version

Reply:
moved